infra/modules/providers/azure/app-monitoring/README.md

```diff
@@ -0,0 +1,92 @@
+# Azure Monitor
+
+This custom Terraform based `app-monitoring` module deploys the "Azure Monitor" service in order to offer visibility into the behavior of your deployed Azure Resources. This module is recommended for monitoring Azure Resources deployed alongside App Services (ex. App Service Plan, App Gateway, VM, etc.). For direct monitoring of an App Service, the [app-insights](https://github.com/microsoft/cobalt/tree/master/infra/modules/providers/azure/app-insights) module is instead recommended. That module leverages an "Azure Monitor" service called [application insights](https://www.terraform.io/docs/providers/azurerm/r/application_insights.html).
```

Comment:
A few nits, but overall this is a fantastic intro to the module!
 - Since everything in the repo is custom, I think we can omit that word
 - You can use a relative link to the app-insights module like so: `[app-insights](../../path-to-module)` (the path is made up here). That way it will reflect any changes in your PR if you decide to change that module, and will hold up if the repo name ever changes or the module is moved to another repo

Reply:
Relative path was a great suggestion. Line 14 is where the correction was made.